optimistic-transaction.md

```diff
@@ -7,35 +7,36 @@ aliases: ['/docs/dev/reference/transactions/transaction-optimistic/','/docs/dev/
 
 # TiDB Optimistic Transaction Model
 
-This document introduces the principles of TiDB's optimistic transaction model. This document assumes that you have a basic understanding of [TiDB architecture](/architecture.md), [Percolator](https://ai.google/research/pubs/pub36726), and the [ACID](/glossary.md#acid) properties of transactions.
+This document introduces the principles of TiDB's optimistic transaction model and related features.
 
-In TiDB's optimistic transaction model, the two-phase commit begins right after the client executes the `COMMIT` statement. Therefore, the write-write conflict can be observed before the transactions are actually committed.
+TiDB uses the optimistic transaction model by default. In TiDB's optimistic transaction model, the two-phase commit begins only after the transaction is committed.
 
 > **Note:**
 >
-> Starting from v3.0.8, TiDB uses the [pessimistic transaction model](/pessimistic-transaction.md) by default. However, this does not affect your clusters if you upgrading from v3.0.7 or earlier to v3.0.8 (and later). In other words, **only newly created clusters default to using the pessimistic transaction model**.
+> Starting from v3.0.8, the newly created TiDB clusters use the [pessimistic transaction model](/pessimistic-transaction.md) by default. However, this does not affect your clusters if you upgrading from v3.0.7 or earlier to v3.0.8 (and later). In other words, **only newly created clusters default to using the pessimistic transaction model**.
 
 ## Principles of optimistic transactions
 
-TiDB adopts Google's Percolator transaction model, a variant of two-phase commit (2PC) to ensure the correct completion of a distributed transaction. The procedure is as follows:
+To support distributed transactions, TiDB adopts two-phase commit (2PC) in the optimistic transaction model. The procedure is as follows:
 
 ![2PC in TiDB](/media/2pc-in-tidb.png)
 
 1. The client begins a transaction.
 
-    TiDB receives the start version number (monotonically increasing in time and globally unique) from PD and mark it as `start_ts`.
+    TiDB receives the timestamp (monotonically increasing in time and globally unique) from PD as the unique transaction ID of the current transaction, which is called `start_ts`. TiDB implements multi-version concurrency control, so `start_ts` also serves as the database snapshot version obtained by this transaction. The transaction can only read the data available for the `start_ts` version.
```

Comment:
```suggestion
    TiDB gets a timestamp (monotonically increasing in time and globally unique) from PD as the unique transaction ID of the current transaction, which is called `start_ts`. TiDB implements multi-version concurrency control, so `start_ts` also serves as the version of the database snapshot obtained by this transaction. This means that the transaction can only read the data from the database at `start_ts`.
```
PTAL at my suggestion and see if it's technically correct. @jackysp

Reply:
It looks OK.